src/blockfilter.cpp

```diff
@@ -81,7 +81,7 @@ GCSFilter::GCSFilter(const Params& params)
 GCSFilter::GCSFilter(const Params& params, std::vector<unsigned char> encoded_filter)
     : m_params(params), m_encoded(std::move(encoded_filter))
 {
-    VectorReader stream(GCS_SER_TYPE, GCS_SER_VERSION, m_encoded, 0);
+    SpanReader stream(GCS_SER_TYPE, GCS_SER_VERSION, m_encoded, 0);
```

Comment:
```suggestion
    SpanReader stream{GCS_SER_TYPE, GCS_SER_VERSION, m_encoded, 0};
```

nit: Haven't checked if it compiles.

Reply:
Done, everywhere.